lib/shared/config-manager/src/ConfigSource.ts

```diff
@@ -10,20 +10,31 @@ export abstract class ConfigSource {
      * Should return null if the config has not changed, and throw an error if it could not be retrieved.
      * @param sdkKey
      * @param kind
+     * @param obfuscated
      * @param lastModifiedThreshold
      */
     abstract getConfig(
         sdkKey: string,
         kind: 'server' | 'bootstrap',
+        obfuscated: boolean,
         lastModifiedThreshold?: string,
```

Comment:
I wonder if we should just move all these params into a shared type rather than keep passing them all around?